docker/cmd/run.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+chmod +x run.sh
```

Comment:
Ahhhh...sorry that I made you misunderstand. I mean `chmod` this file `docker/cmd/run.sh` to make it executable. This way the shebang is recognized when `./run.sh` is run,  and `/bin/bash` will be used to interpret `run.sh`